In [17]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

df = pd.read_csv('dinkes-od_17153_jml_unit_transfusi_darah__kabupatenkota_v2_data.csv')
df

id  kode_provinsi nama_provinsi  kode_kabupaten_kota  \
0      1             32    JAWA BARAT                 3201   
1      2             32    JAWA BARAT                 3202   
2      3             32    JAWA BARAT                 3203   
3      4             32    JAWA BARAT                 3204   
4      5             32    JAWA BARAT                 3205   
..   ...            ...           ...                  ...   
157  158             32    JAWA BARAT                 3275   
158  159             32    JAWA BARAT                 3276   
159  160             32    JAWA BARAT                 3277   
160  161             32    JAWA BARAT                 3278   
161  162             32    JAWA BARAT                 3279   

    nama_kabupaten_kota  jumlah_utd satuan  tahun  
0       KABUPATEN BOGOR           1   UNIT   2019  
1    KABUPATEN SUKABUMI           1   UNIT   2019  
2     KABUPATEN CIANJUR           1   UNIT   2019  
3     KABUPATEN BANDUNG           1   UNIT   2019  
4       KABUPATEN GARUT           1   UNIT   2019  
..                  ...         ...    ...    ...  
157         KOTA BEKASI           1   UNIT   2024  
158          KOTA DEPOK           1   UNIT   2024  
159         KOTA CIMAHI           1   UNIT   2024  
160    KOTA TASIKMALAYA           2   UNIT   2024  
161         KOTA BANJAR           1   UNIT   2024  

[162 rows x 8 columns]

<li>🔢 Statistik Umum
Total jumlah UTD: 153 unit

Rata-rata jumlah UTD per kabupaten/kota: ~0.94 unit

Jumlah data kabupaten/kota: 162 entri

🏆 Kabupaten/Kota dengan Jumlah UTD Tertinggi
Nama: Kabupaten Bandung

Jumlah UTD: 5 unit

Provinsi: Jawa Barat

Tahun: 2024

⚠️ Kabupaten/Kota dengan Jumlah UTD Terendah
Nama: Kabupaten Pangandaran

Jumlah UTD: 0 unit

Provinsi: Jawa Barat

Tahun: 2019

🗺️ Distribusi UTD per Provinsi
Provinsi dengan total UTD terbanyak:

Jawa Barat: 153 unit (satu-satunya provinsi dalam data ini)

Jumlah kabupaten/kota di provinsi tersebut: 162
(Menunjukkan bahwa data ini hanya mencakup Provinsi Jawa Barat)

📝 Kesimpulan Sementara
Data hanya mencakup Provinsi Jawa Barat, dengan 162 kabupaten/kota (termasuk kota administratif) tercatat.

Jumlah UTD bervariasi dari 0 hingga 5 per kabupaten/kota.

Mayoritas daerah hanya memiliki 1 atau 0 unit UTD, menunjukkan ketimpangan distribusi yang signifikan.

Perlu ditelusuri lebih lanjut bagaimana tren data dari tahun ke tahun jika tersedia, serta bagaimana hubungan antara jumlah penduduk dan kebutuhan UTD di masing-masing daerah.</li>